File: src/Client/inventory-track/src/app/models/dto/writeoff-request/create-writeoff-request-dto.ts
// src/app/models/dto/writeoff-request/create-writeoff-request-dto.ts
export interface CreateWriteOffRequestDto {
  itemId: string;
  warehouseId: string;
  quantity: number;
  reasonId?: string | null;
  anotherReason?: string;
  companyId:string;
}

File: src/Services/WriteOffService/WriteOffService.Application/DTOs/Request/WriteOffRequest/CreateWriteOffRequestDto.cs
using Microsoft.AspNetCore.Http;
using WriteOffService.Domain.Entities;

namespace WriteOffService.Application.DTOs.Request.WriteOffRequest;

public class CreateWriteOffRequestDto
{
    public Guid ItemId { get; set; }
    public Guid WarehouseId { get; set; }
    public long Quantity { get; set; }
    public Guid? ReasonId { get; set; }
    public string? AnotherReason { get; set; }
    public List<IFormFile> Documents { get; set; } = new();
    public Guid CompanyId { get; set; }
}